src/options.html

```diff
@@ -52,18 +52,55 @@ <h6 class="text-secondary">status of the tasks</h6>
         <h4>Sources</h4>
         <h6 class="text-secondary">sources of the tasks</h6>
         <div class="d-flex flex-column pl-4">
+          <div>
+            <input
+              class="selectallsource"
+              type="checkbox"
+              name="allSelector"
+              id="selectAll"
+            />
+            <label> Select All </label>
+          </div>
           <div>
             <input
               class="sources"
               type="checkbox"
               name="Programming.in.th"
               id="programming.in.th"
             />
-            <label for="programming.in.th">
-              Programming.in.th
+            <label for="programming.in.th"> Programming.in.th </label>
+            <br />
+            <input
+              class="subSource"
+              type="checkbox"
+              name="EasyProgramming.in.th"
+              id="pgEasy"
+            />
+            <label>
+              Easy
               <span class="badge badge-info ml-1">00..</span>
+            </label>
+            <br />
+            <input
+              class="subSource"
+              type="checkbox"
+              name="EasyProgramming.in.th"
```

Comment:
Please change the value of `name` tag to make it correspond with the `id`.

Reply:
I just fixed it. Can you review it again?